examples/rules.kts

```diff
@@ -40,7 +40,7 @@ val copyleftLimitedLicenses = getLicenseSet("copyleft-limited")
 val publicDomainLicenses = getLicenseSet("public-domain")
 
 // The complete set of licenses covered by policy rules.
-val handledLicenses = listOf(
+val handledLicenses = setOf(
```

Comment:
I believe this has no effect as the `flatten()` in line 48 will convert this to a list. But that does not matter anyway, as in line 54 we already convert the whole thing to a `Set`, so we should be fine as-is.

BTW, we compile the examples as part of our PR checks, so we should get notified of any breakages. Phrased differently, the example should be fine in its current state.